kolibri/plugins/management/assets/src/state/manageContentActions.js

```diff
@@ -68,70 +67,81 @@ function addChannelFileSummaries(store, channelIds) {
  * @param {Object} params - data needed to execute transition
  * @returns {undefined}
  */
-function transitionWizardPage(store, transition, params) {
+export function transitionWizardPage(store, transition, params) {
```

Comment:
I like these individual `export`s placed throughout the file. Reduces boilerplate compared to the single `export` object at the end.